pkg/storage/queue.go

```diff
@@ -531,6 +531,10 @@ func (bq *baseQueue) processReplica(
 				log.VEventf(queueCtx, 3, "not holding lease; skipping")
 				return nil
 			}
+			if _, harmless := err.GetDetail().(*roachpb.RangeNotFoundError); harmless {
+				log.Event(queueCtx, "range not found on node; skipping")
```

Comment:
3 seems like a pretty infrequent verbosity for something that only comes up while replicas are being GC'ed, not all the time like the above case. I've switched it though.